src/App.jsx

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import PrivateRoute from './components/PrivateRoute/PrivateRoute';
+import { AuthProvider } from './context/AuthContext';
+import Login from './views/Auth/Login';
 import Home from './views/Home/Home';
 import Layout from './views/Layout/Layout';
 
 export default function App() {
   return (
-    <Layout>
-      <Home />
-    </Layout>
```

Comment:
Great Job here adding your context provider around your app to access your state.